toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -890,6 +898,12 @@ The path of the script.
 This must be in the same directory or a sub-directory that the config file is located
 in.
 
+Mutually exclusive with `content`.
+
+If `interpreter` is not specified, then the script file is executed directly.
+Otherwise, the script file's path is passed as the first parameter to the `interpreter`
```

Comment:
nit - it is necessary to mention "first" parameter? might be confusing to user who is not familiar with our functionality